CMakeLists.txt

```diff
@@ -279,4 +279,5 @@ add_custom_target(clean-all
 # installation
 
 install(TARGETS parquet
-  LIBRARY DESTINATION lib)
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib)
```

Comment:
This is to enable static builds. Not relevant to parquet-502, but useful in general, and not worth opening a separate jira.